uber/custom_tags.py

```diff
@@ -445,10 +445,12 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
+        skiprest = ''
         for day, price in BADGE_PRICES['single_day'].items():
             if day == datetime.strftime(ESCHATON, "%A"):
                 prices += 'and ${} for {}'.format(price, day)
-            else:
```

Comment:
Hm, instead of this, can you use a `break` statement here to exit out of the for loop?